pymc3/distributions/logprob.py

```diff
@@ -302,10 +257,13 @@ def incsubtensor_logp(op, var, rvs_to_values, indexed_rv_var, rv_values, *indice
     return logp_var
 
 
-@_logp.register(Subtensor)
-@_logp.register(AdvancedSubtensor)
-@_logp.register(AdvancedSubtensor1)
-def subtensor_logp(op, var, rvs_to_values, indexed_rv_var, *indices, **kwargs):
+@_logprob.register(Subtensor)
```

Comment:
For some reason these aren't being registered onto `_logprob`.

Reply:
We shouldn't need these anymore, right?  The functionality should already be covered by AePPL.